app/uk/gov/hmrc/cataloguefrontend/costs/CostExplorerController.scala

```diff
@@ -43,13 +51,53 @@ class CostsSummaryController @Inject() (
 ) extends FrontendController(mcc)
     with CatalogueAuthBuilders {
 
-  def costExplorer(team: Option[String] = None): Action[AnyContent] =
-    BasicAuthAction.async { implicit request =>
+  private def toRows(serviceDeploymentMap: Map[String, Seq[DeploymentConfig]]): Seq[Seq[(String, String)]] = {
+    serviceDeploymentMap.map {
+      case (serviceName, configList) =>
+
+        val configEstimateCosts = String.format("%.0f", configList.map(_.deploymentSize.totalSlots.costGbp(costEstimateConfig)).sum)
+
+        val slotsAndInstances = Environment.values.flatMap {
+          env =>
+            Seq(
+              s"slots.{${env.asString}}" -> configList.find(_.environment == env).map(_.deploymentSize.slots.toString).getOrElse(""),
+              s"instances.{${env.asString}}" -> configList.find(_.environment == env).map(_.deploymentSize.instances.toString).getOrElse("")
+            )
+        }
+
+        Seq("Application Name" -> serviceName) ++ slotsAndInstances ++ Seq("Estimated Cost (£ / year)" -> configEstimateCosts)
+    }.toSeq
+  }
 
+  def costExplorer(
+    name: Option[String] = None,
```

Comment:
do we need the `name` filter? It's misleading since it only works if `asCSV` is true and CSVs can be filtered as required once exported.